srcpkgs/antiword/patches/10_fix_buffer_overflow_wordole_c.patch

```diff
@@ -3,7 +3,7 @@ Author: <eriks@debian.org>
 Bug-Debian: http://bugs.debian.org/407015
 Last-Update: 2009-06-03
 
---- wordole.c	2005-08-26 21:49:57.000000000 +0200
+--- antiword-0.37/wordole.c	2005-08-26 21:49:57.000000000 +0200
 +++ wordole.c	2009-06-03 22:31:15.948014682 +0200
```

Comment:
pretty sure this is wrong, as it will patch the file into a wrong location, both `---` and `+++` need to be changed

Reply:
hm, or maybe not - it seems to be behaving correctly - but it's ugly either way